windows/src/developer/inst/copydev.in

```diff
@@ -2,22 +2,22 @@
 
 copykmdev: makeinstaller
     -mkdir $(ROOT)\release\$VERSION
-    copy /Y $(ROOT)\src\developer\inst\keymandeveloper100.msi $(ROOT)\release\$VERSION\keymandeveloper100.msi
+    copy /Y $(ROOT)\src\developer\inst\keymandeveloper.msi $(ROOT)\release\$VERSION\keymandeveloper.msi
     copy /Y $(ROOT)\src\developer\inst\keymandeveloper-$VERSION.exe $(ROOT)\release\$VERSION\keymandeveloper-$VERSION.exe
```

Comment:
granted you're updating the installer in another PR.

This looks similar to keymandesktop.exe above, where `-$VERSION` goes away?

Reply:
Dup response: No, we maintain the version number in the .exe, just remove in lower level files. ;-)